### PR TITLE
Adds eslint and bumps version

### PR DESCRIPTION
**Pull request by abea**
See comments below for changes that aren't simply white space.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -160,7 +163,7 @@ function sanitizeHtml(html, options, _recursing) {
       stack.push(frame);
 
       var skip = false;
-      var hasText = frame.text ? true : false;
+      var hasText = !!frame.text;
```
Comment (abea) [inline src/index.js]: Not simply white space. Auto fix.

File: test/test.js [hunk 2]
```diff
@@ -392,7 +428,7 @@ describe('sanitizeHtml', function() {
             }
           },
           exclusiveFilter: function(frame) {
-            return frame.tag === 'a' && frame.text.trim() == 'blah';
+            return frame.tag === 'a' && frame.text.trim() === 'blah';
```
Comment (abea) [inline test/test.js]: Not simply white space.

File: test/test.js [hunk 3]
```diff
@@ -364,12 +399,12 @@ describe('sanitizeHtml', function() {
...
```
Comment (abea) [inline test/test.js]: Not simply white space.

File: src/index.js (hunk 1 above)
Comment (boutell) [reply src/index.js]: Aha! Told everyone this convention wasn't just in my head 😆

**Pull request merged**